pyramid_openapi3/check_openapi_responses.py

```diff
@@ -0,0 +1,89 @@
+"""Verify that endpoints have defined required responses."""
+
+from .exceptions import ResponseValidationError
+from openapi_core.schema.specs.models import Spec
+from pathlib import Path
+from pyramid.config import Configurator
+
+import openapi_core
+import typing as t
+import yaml
+
+
+def get_config(config_path: str) -> t.Dict[str, str]:
+    """Read config from file."""
+    config = Path(config_path)
+
+    if not config.is_file():
+        raise Exception(f"ERROR Config file not found on: {config}\n")
+
+    with open(config, "r") as f:
+        return yaml.safe_load(f)
+
+
+def get_spec(schema_path: str) -> Spec:
+    """Create openapi spec from schema."""
+    schema = Path(schema_path)
+
+    if not schema.is_file():
+        raise Exception(f"ERROR schema file not found on: {schema}\n")
+
+    with open(schema, "r") as f:
+        return openapi_core.create_spec(yaml.safe_load(f))
+
+
+def required_responses(
+    config: t.Dict, endpoint: str, method: str, has_params: bool
+) -> t.Set:
+    """Get required responses for given method on endpoint."""
+    required_resp: t.Set = set(config.get("required_responses", {}).get(method, []))
+    if has_params:
+        required_params: t.Set = set(
+            config.get("required_responses_params", {}).get(method, [])
+        )
+        required_resp = required_resp.union(required_params)
+    allowed_missing: t.Set = set(
+        config.get("allowed_missing_responses", {}).get(endpoint, {}).get(method, [])
+    )
+    required_resp = required_resp - allowed_missing
+    return required_resp
+
+
+def validate_required_responses(spec_file: str, config: Configurator) -> None:
+    """Verify that all endpoints have defined required responses."""
+    check_failed: bool = False
+    missing_responses_count: int = 0
+    errors: t.List = []
```

Comment:
```suggestion
    errors =  []
```
Is also fine, since this is not init function and variable is not publicly accessible.

Reply:
Done.